libnmstate/state.py

```diff
@@ -45,35 +45,78 @@
 from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Route
+from libnmstate.schema import RouteRule
 
 
 NON_UP_STATES = (InterfaceState.DOWN, InterfaceState.ABSENT)
 
 
 @total_ordering
-class RouteEntry(object):
+class _StateObject(object):
+    def __init__(self):
+        self._absent = False
+
+    def keys(self):
+        return ()
+
+    def __hash__(self):
+        return hash(self.keys())
+
+    def __eq__(self, other):
+        return self is other or self.keys() == other.keys()
+
+    def __lt__(self, other):
+        return self.keys() < other.keys()
+
+    def __repr__(self):
+        return str(self.to_dict())
+
+    @property
+    def absent(self):
+        return self._absent
```

Comment:
I do not understand what it is supposed to represent. Is it a question (is_absent)? Is it supposed to return the state of the current entry as absent?